server/handle_policy_eval.go

```diff
@@ -0,0 +1,163 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package server
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/eclipse-xpanse/policy-man/config"
+	"github.com/eclipse-xpanse/policy-man/log"
+	"github.com/gin-gonic/gin"
+	"github.com/gin-gonic/gin/binding"
+	"github.com/open-policy-agent/opa/rego"
+	"net/http"
+	"runtime/debug"
+)
+
+type EvalRego struct {
+	Policy  string `json:"policy" binding:"required"`
+	IsAllow bool   `json:"isAllow" binding:"required"`
```

Comment:
AFAIK, we don't need this flag. We must evaluate the full rego file/files based on the input. Whether the response is allow or deny is the decided by the policy eval method.